deploy/charts/README.md

```diff
@@ -111,60 +111,60 @@ helm install openebs-monitoring openebs-monitoring/monitoring --namespace openeb
+| `openebsMonitoringAddon.jiva.serviceMonitor.selector`                                     | Selector to select endpoints objects                                                | `{matchLabels: {openebs.io/cas-type: jiva}}`              |
+| `openebsMonitoringAddon.jiva.serviceMonitor.namespaceSelector`                            | Selector to select which namespaces the endpoints objects are discovered from       | `[any: true]`                                             |
+| `openebsMonitoringAddon.lvmLocalPV.serviceMonitor.enabled`                                | Enables monitoring of lvm localPV                                                   | `true`                                                    |
+| `openebsMonitoringAddon.lvmLocalPV.serviceMonitor.endpoints.port`                         | Name of the service port lvm localPV node endpoint refers to                        | `"metrics"`                                               |
+| `openebsMonitoringAddon.lvmLocalPV.serviceMonitor.endpoints.path`                         | HTTP path to scrape for metrics from lvm localPV node                               | `"/metrics"`                                              |
+| `openebsMonitoringAddon.lvmLocalPV.serviceMonitor.selector`                               | Selector to select endpoints objects                                                | `{matchLabels: {name: openebs-lvm-node}}`                 |
+| `openebsMonitoringAddon.lvmLocalPV.serviceMonitor.namespaceSelector`                      | Selector to select which namespaces the endpoints objects are discovered from       | `[any: true]`                                             |
+| `openebsMonitoringAddon.ndm.enabled`                                                      | Enables installation of monitoring addons for OpenEBS ndm                           | `true`                                                    |
+| `openebsMonitoringAddon.ndm.serviceMonitor.enabled`                                       | Enables monitoring of OpenEBS ndm                                                   | `true`                                                    |
+| `openebsMonitoringAddon.ndm.serviceMonitor.endpoints.port`                                | Name of the service port ndm exporters endpoint refers to                           | `"metrics"`                                               |
+| `openebsMonitoringAddon.ndm.serviceMonitor.endpoints.path`                                | HTTP path to scrape for metrics from ndm exporters                                  | `"/metrics"`                                              |
+| `openebsMonitoringAddon.ndm.serviceMonitor.selector`                                      | Selector to select endpoints objects                                                | `{matchLabels: {app: openebs-ndm-exporter}}`              |
```

Comment:
additional whitespace at the end can be removed.

Reply:
It is automatically added when I format the file.